web/src/components/dashboard/SidebarCardFormationNudge.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { SidebarCardFormationNudge } from "@/components/dashboard/SidebarCardFormationNudge";
+import { SidebarCardContent } from "@/lib/types/types";
+import {
+  generatePreferences,
+  generateProfileData,
+  generateSidebarCardContent,
+  generateUserData,
+} from "@/test/factories";
+import { useMockRouter } from "@/test/mock/mockRouter";
+import {
+  currentUserData,
+  setupStatefulUserDataContext,
+  WithStatefulUserData,
+} from "@/test/mock/withStatefulUserData";
+import { UserData } from "@businessnjgovnavigator/shared/userData";
+import { fireEvent, render, screen } from "@testing-library/react";
+
+jest.mock("@/lib/data-hooks/useUserData", () => ({ useUserData: jest.fn() }));
+jest.mock("next/router", () => ({
+  useRouter: jest.fn(),
+}));
+
+describe("<SidebarCardFormationNudge />", () => {
+  let card: SidebarCardContent;
+
+  const renderWithUserData = (userData: Partial<UserData>) => {
+    render(
+      <WithStatefulUserData initialUserData={generateUserData(userData)}>
+        <SidebarCardFormationNudge card={card} />
+      </WithStatefulUserData>
+    );
+  };
+
+  describe("when clicking funding button", () => {
+    jest.resetAllMocks();
+    useMockRouter({});
+    setupStatefulUserDataContext();
+    card = generateSidebarCardContent({ id: "formation-nudge" });
+
```

Comment:
these couple lines should be within a `beforeEach` block. Because you have them floating at the top of a `describe`, they will only get run once. This *happens* to work right now because you only have one test in this describe block, if you or someone else were to add another one, having these free-floating at the top of the describe would cause some unexpected behavior.